definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -1686,7 +1686,7 @@
     "Name": "Review documents",
     "Description": "Review documents uploaded",
     "DisplayOrder": 48,
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS;SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```

Comment:
Is this event required for AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS?

Reply:
@yogendraU removed AWAITING_SUBMISSION_TO_HMCTS though this event is required in AWAITING_RESUBMISSION_TO_HMCTS as per 
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1721438715